### PR TITLE
issue #543

### PR DESCRIPTION
**Pull request by meguiraun**
- New Sample modal: focus on sample name by default, ENTER sends addAndMount
- Form validation: at this moment, only checks if name and acronym are supplied

----
Tomorrow and Friday are days off in Sweden, so I will not be able to answer quickly.

### CODE REVIEW

File: mxcube3/ui/components/Tasks/validate_add_sample.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const validate = (values, props) => {
+  const errors = {};
+  if (!props) {
+    // for some reason redux-form is loaded before the initial status @##@!
```
Comment (marcus-oscarsson) [inline mxcube3/ui/components/Tasks/validate_add_sample.js]: What does @##@! mean ?

Comment (meguiraun) [inline mxcube3/ui/components/Tasks/validate_add_sample.js]: some bad work I did not date to write... I copied `./Tasks/validate.js`

File: mxcube3/ui/components/Tasks/validate.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const everpolate = require('everpolate');
 const validate = (values, props) => {
   const errors = {};
   if (!props.attributes) {
-    // for some reason redux-form is loaded before the initial status @##@!
+    // for some reason redux-form is loaded before the initial status
```
Comment (mguijarr) [inline mxcube3/ui/components/Tasks/validate.js]: lol

**Comment by meguiraun** [comment]
Sorry but I am confused... Is there anything else I should fix for this PR?

**Comment by marcus-oscarsson** [comment]
Its great, thanks !

**Comment by meguiraun** [comment]
tack!

**Pull request merged**